Bottom: 67791b659f2c9260281ad3fe339d6cd4b32b52b1
Top:    ca6c089dd94aa5b670fff28aa2f12b0f98b5c628
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 17:37:57 +0000

Mark segments as not round before falling back to line-line intersection

This avoids assert failures with debugging enabeled


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 58ca3fa..32d0ef7 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1292,6 +1292,8 @@ seg_in_seg_arc_arc (struct info *i, struct seg *s1, struct seg *s2)
       s1->v->radius == s2->v->radius)
     {
       printf ("Cop-out for co-circular arcs for now - they will be better handled by their line approximations. <<<<<<<<<<\n");
+      s1->v->is_round = false;
+      s2->v->is_round = false;
       return seg_in_seg_line_line (i, s1, s2);
     }
 #endif
